test.rb

```diff
@@ -0,0 +1,21 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+l = lambda {|a| a }
```

Comment:
__[development][rubocop]__
* Use the `-> { ... }` lambda literal syntax for single line lambdas. [:ref](https://github.com/bbatsov/ruby-style-guide#lambda-multi-line)
* Space between { and | missing.
* Shadowing outer local variable - `a`.